website/src/models/group.js

```diff
@@ -164,7 +175,7 @@ GroupSchema.methods.sendChat = function(message, user){
   // Kick off chat notifications in the background.
   var lastSeenUpdate = {$set:{}, $inc:{_v:1}};
   lastSeenUpdate['$set']['newMessages.'+group._id] = {name:group.name,value:true};
-  if (NO_CHAT_NOTIFICATIONS.indexOf(group._id) !== -1 || group.memberCount > 5000) {
+  if (NO_CHAT_NOTIFICATIONS.indexOf(group._id) !== -1 || group.isTooLargeForChatNotifications) {
```

Comment:
The NO_CHAT_NOTIFICATIONS variable was deleted above, right? Should probably remove the indexOf or keep the variable above.